stormforge/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "6f1ddcc9-e704-4f94-941b-8a914fcd89a0",
+    "app_id": "stormforge",
+    "display_on_public_website": true,
+    "tile": {
+        "overview": "README.md#Overview",
+        "configuration": "README.md#Setup",
+        "support": "README.md#Support",
+        "changelog": "CHANGELOG.md",
+        "description": "Real-time optimization of Kubernetes resource settings using machine learning.",
+        "title": "StormForge",
+        "media": [{
+                "media_type": "image",
```

Comment:
Note that these images do not have a 16:9 aspect ratio, so they will get cut off in the preview of the media carousel (the full image will be shown once a user clicks on it). If you'd like to avoid this, feel free to paste the images on a background with a 16:9 aspect ratio.

Reply:
Will work on getting the media sorted.